metal/resource_metal_spot_market_request_test.go

```diff
@@ -17,8 +17,8 @@ resource "metal_project" "test" {
 }
 
 data "metal_spot_market_price" "test" {
-  facility = "sv15"
-  plan     = "c3.small.x86"
+  facility = "ewr1"
```

Comment:
sv15 would be preferable here. I think c3.small would be preferable as well.

Reply:
replaced with c3.medium